.github/workflow/pythonpackage.yml

```diff
@@ -0,0 +1,80 @@
+# This workflow will check package installation and runs tests
+# On Ubuntu, Windows and MacOS
+# Triggered by push to master
+
+name: Test package
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  installation:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install poetry
```

Comment:
Why we are not using github actions for this? -> https://github.com/marketplace/actions/setup-poetry

Reply:
I had issues with two actions for poetry installation for different os. Moreover, I suppose, this action is not official one, so pip installation looks more reliable

Reply:
Ok, then we'll use this workaround till release of the official action or at least semi-official one.